docs/main/BasicClient/AwaitedDOM-Extensions.md

```diff
@@ -106,7 +93,35 @@ Attached to Nodes and Elements ([see list](#super-nodes)).
 
 #### **Returns**: `Promise<void>`
 
-### node.$type *(...typeInteractions)* {#type}
+
+### element.$detach *(name?)* {#extract-later}
+
+Detaches an element from the browser's live [AwaitedDOM](/docs/hero/basic-client/awaited-dom) environment and converts it into a [DetachedDOM](/docs/hero/basic-client/detached-dom) object for local usage (i.e, without any need for promises or awaits). You can optionally supply a name as the first argument if you want to add it to the [hero.detachedElements](/docs/hero/basic-client/hero#detached-elements) bucket. The advantage of hero.detachedElements is you can use and reuse them from within [HeroReplay(/docs/hero/basic-client/hero-replay) long after your Hero session has closed. This allows you to write extraction logic that can be easily iterated on without needing to reload the webpage(s).
+
+For example, below is a simple hero script that collects the `h1` element:
+```js
+const hero = new Hero();
+await hero.goto('https://ulixee.org');
+await hero.querySelector('h1').$detach('title');
+console.log('Session ID: ', await hero.sessionId);
+```
+
+You can create a second script that uses [HeroReplay](/docs/hero/basic-client/hero-replay) to find the data you need without loading the website again:
+```js
+const hero = new HeroReplay({ /* previousSessionId */});
+const h1 = await hero.detachedElements.get('title');
+const h1Children = [...h1.querySelectorAll('div')].map(x => x.textContent);
+```
+
+It's important to remember that the returned element returned is no longer [AwaitedDOM](/docs/hero/basic-client/awaited-dom). It is in the form of [DetachedDOM](/docs/hero/basic-client/detached-dom), which allows access to properties and methods without the `await` keyword.
```

Comment:
Why is it important? Doesn't it work with or without the awaits?

Reply:
Yeah, the wording was too much. I simplified and removed the "It's important".